rllab/envs/grid_world_env.py

```diff
@@ -1,34 +1,18 @@
+from gym import Env
 import numpy as np
-from .base import Env
+
 from rllab.spaces import Discrete
 from rllab.envs import Step
+from rllab.envs.env_spec import EnvSpec
 from rllab.core import Serializable
 
 MAPS = {
-    "chain": [
```

Comment:
The original formatting was clearly better, so you can use # yapf: disable on `MAPS` to keep YAPF from complaining.